.github/workflows/docker-test.yml

```
@@ -0,0 +1,15 @@
+name: Build and Push Subgraph Deployer Docker Image
```

```suggestion
name:Dry Run Deployer Docker Image
```
May be we can rename the file also as dry run

---

It's rename to `Test Docker Image`: https://github.com/iExecBlockchainComputing/iexec-voucher-subgraph/pull/36/files#diff-baa08c0b7cb0ae6357ecf7ecaa2d7aff0bfa417e10e1ec6fbc29232cb0166da7R1